docs_app/content/guide/overview.md

```diff
@@ -1,127 +1,50 @@
 # Introduction
 
-RxJS is a library for composing asynchronous and event-based programs by using observable sequences. It provides one core type, the [Observable](./guide/observable), satellite types (Observer, Schedulers, Subjects) and operators inspired by [Array#extras](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.6) (map, filter, reduce, every, etc) to allow handling asynchronous events as collections.
-
-<span class="informal">Think of RxJS as Lodash for events.</span>
-
-ReactiveX combines the [Observer pattern](https://en.wikipedia.org/wiki/Observer_pattern) with the [Iterator pattern](https://en.wikipedia.org/wiki/Iterator_pattern) and [functional programming with collections](http://martinfowler.com/articles/collection-pipeline/#NestedOperatorExpressions) to fill the need for an ideal way of managing sequences of events.
-
-The essential concepts in RxJS which solve async event management are:
-
-- **Observable:** represents the idea of an invokable collection of future values or events.
-- **Observer:** is a collection of callbacks that knows how to listen to values delivered by the Observable.
-- **Subscription:** represents the execution of an Observable, is primarily useful for cancelling the execution.
-- **Operators:** are pure functions that enable a functional programming style of dealing with collections with operations like `map`, `filter`, `concat`, `reduce`, etc.
-- **Subject:** is the equivalent to an EventEmitter, and the only way of multicasting a value or event to multiple Observers.
-- **Schedulers:** are centralized dispatchers to control concurrency, allowing us to coordinate when computation happens on e.g. `setTimeout` or `requestAnimationFrame` or others.
-
-## First examples
-
-Normally you register event listeners.
-
-```ts
-document.addEventListener('click', () => console.log('Clicked!'));
-```
-
-Using RxJS you create an observable instead.
+RxJS is a library implementing the reactive programming paradigm for Javascript. This programming paradigm mainly handles the propagation of change. In reactive programming, one specifies how to "react" on such a propagated change. Taking a look at the following code example clarifies the behavior of reactive programming. 
 
 ```ts
-import { fromEvent } from 'rxjs';
+let a = 10;
+let b = 10;
+let c = a + b;
+a = 20;
 
-fromEvent(document, 'click').subscribe(() => console.log('Clicked!'));
+console.log(c);
 ```
 
-### Purity
+Generally, considering usual programming paradigms the code above will log `20`. In a fully reactive approach the code would consider the change in the variable `a` and therefore log `30` instead. 
 
-What makes RxJS powerful is its ability to produce values using pure functions. That means your code is less prone to errors.
+This is only an example demonstrating the behavior of reactive programming. 
```

Comment:
This file has also couple of empty spaces at the end of some lines. If this is not important, please let me know, I won't report it anymore.

Reply:
I don't have really strong opinions about that. IMO it helps maintaining the markdown file

Reply:
I kinda don't like empty spaces at the end of lines, they look a bit dirty to me. But if they can help, sure, keep having them.